server/src/index.ts

```diff
@@ -168,8 +164,17 @@ app.get("/users", async (req, res) => {
       const radius = Number(req.query.radius);
 
       const userIds = await findNearbyUsers(lat, lon, radius);
-      console.log(userIds);
       res.json(userIds);
+    } else if (req.query.userId) {
+      query = "?userId";
+      const userId = req.query.userId;
+      if (typeof userId != "string") throw Error("  [userId] is not a string.");
+
+      const success = await getConnectedUserDisplayName(userId);
+      if (success) {
+        res.json(success);
```

Comment:
can you rename this to something other than success. the variable name is not descriptive enough for people to know that it is a displayName: "string". You could rename it `displayName`.